server/chat-commands/info.ts

```diff
@@ -2644,6 +2645,32 @@ export const commands: ChatCommands = {
 	},
 	showhelp: [`/show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`],
 
+	regdate: 'registertime',
+	regtime: 'registertime',
+	async registertime(target, room, user) {
+		this.checkChat();
```

Comment:
Is this necessary? It probably shouldn't be for `/` use, and for `!` use it should already be a part of `runBroadcast`, right?

Reply:
I think / use would be practical; but that's just me.

Reply:
I think you're misinterpreting me. I'm saying the user shouldn't need to be able to talk to use `/regdate`.

Reply:
Ah. I think it probably would be prudent, so someone (cough simplechimp cough) can't abuse it to send too many requests in a short period. But i'll defer to you.

Reply:
Does `checkChat` put it on a chat throttle, though? We might want to manually put it on the chat throttle?

Reply:
What's your goal in this? Because I don't believe `checkChat` will prevent simplechimp from abusing it.